src/main/java/mat/shared/model/util/MeasureDetailsUtil.java

```diff
@@ -94,4 +100,22 @@ public static String defaultTypeIfBlank(String type) {
         return type == null || type.isEmpty() ? PRE_CQL : type;
     }
 
+    public static boolean isValidatable(String releaseVersion, String qdmVersion, boolean draft, boolean draftable, String modelType) {
+        if (releaseVersion == null || qdmVersion == null || modelType == null) {
+            return false;
+        }
+        BigDecimal matVersion = null;
+        BigDecimal measureVersion = new BigDecimal(qdmVersion);
+
+        RegExp regExp = RegExp.compile("[0-9]+\\.[0-9]+");
+        MatchResult matcher = regExp.exec(releaseVersion);
+        if (matcher != null) {
+            matVersion = new BigDecimal(matcher.getGroup(0));
+        }
+
+        return (draft && modelType.equals(FHIR))
+                || (draftable && modelType.equals(QDM)
+                && RUN_FHIR_VALIDATION_VERSION.compareTo(matVersion) == -1
```

Comment:
Interesting.. it took me quite a while to realize why ==-1 works for you and not >= 0.
Apparently you use 5.7 and 5.4 instead of 5.8 and 5.5 :)

Reply:
Do I have to use <= 0 instead of -1 ?

Reply:
Not sure about that. Your approach is valid too.

Reply:
Using <= 0 instead of -1